spec/support/test_cluster.rb

```diff
@@ -0,0 +1,142 @@
+#taken from: https://github.com/bsm/promiscuous_cluster/blob/09241015f293bc6056e45af523dd2dce43643d21/scenario/scenario.rb
+
+require 'fileutils'
+require 'pathname'
+
+class TestCluster
```

Comment:
except for commented line, just copied out of promiscuous so we aren't coupled to promiscuous spec helpers